main/pipeline/pipeline.cc

```diff
@@ -1075,6 +1143,24 @@ ast::ParsedFilesOrCancelled typecheck(unique_ptr<core::GlobalState> &gs, vector<
                 opts.print.FileTableJson.print(buf.str());
             }
         }
+        if (opts.print.FileTableMessagePack.enabled) {
+            auto files = core::Proto::filesToProto(*gs);
+            stringstream buf;
+            core::Proto::toJSON(files, buf);
+            auto str = buf.str();
+            rapidjson::Document document;
+            document.Parse(str);
```

Comment:
Is re-parsing the json output a temporary solution?

Reply:
Temporary up-until someone invests in better one :-)